ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ntp.md

```diff
@@ -60,13 +60,22 @@ interface Management1
 | 10.1.1.1 | true |
```

Comment:
you submitted generate artifacts, but the PR doesn't include the augmented test case -> https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ntp.yml